lab-lindsay/.env

```diff
@@ -0,0 +1 @@
+API_URL='http://localhost:3000'
```

Comment:
be careful about pushing up env variables. especially if you have api keys that could be scraped.